### ISSUE TITLE
Skip step

### ISSUE THREAD

**Issue by Yegorov** [braintree/runbook#14]
It would be nice to be able to temporarily skip step. For example, great solution is to add `x` prefix or add `:skip` argument to `step`, as implemented in [rspec](https://relishapp.com/rspec/rspec-core/v/3-9/docs/pending-and-skipped-examples/skip-examples#temporarily-skipping-by-prefixing-%60it%60,-%60specify%60,-or-%60example%60-with-an-x)
If you think this is a great idea, I would gladly send PR, with implementation this feature.

**Comment by Yegorov** [comment]
Hello @pblesi!
Thanks, for answering.

The main idea is that the step that is still in development should not be performed at launch time, 
but it can be seen during rendering. If we comment a step, then it will completely disappear during execution and rendering runbook.

**Comment by pblesi** [comment]
My concern is that some may want skip to mean skip only during execution and others may want skip to mean skip during execution _and_ rendering. I'm going to take a stab at adding some functionality that should enable what you need. I'll PR it for review once it's complete

**Comment by pblesi** [comment]
I've created the following branch that adds the concept of tags and labels: 

https://github.com/braintree/runbook/compare/master...add_entity_tags

It includes an example for how to add a hook to skip execution of any steps or sections that include the `:skip` tag (based on the implementation, the entities will not be skipped for views).

Some other additions that could be made (though I'm not convinced they should be made at this point in time unless there is sufficient interest):

* Include the skip hook for `:skip` tags by default

* Add `--tag` command line argument analogous to [rspec's tags](https://relishapp.com/rspec/rspec-core/docs/command-line/tag-option)

* Add `xsection` and `xstep` functionality
  * It is not clear if this would skip just the run, or the run and the view.
  * Skipping both would be an easy implementation.
  * Skipping just the run or view, would be more complex but doable. 

I will look to merge this before Runbook's next release. I'm curious to hear your thoughts.

**Comment by Yegorov** [comment]
Ok, thanks for this feature. In my opinion, great solution to add skip hook for :skip tags by default. Other add-ons are not important right now

**Comment by pblesi** [comment]
Labels and tags have been added in release (0.16.0). I'm still weary to add the skip hook outright as I'm not convinced the desired behavior for skipping an entity is well-defined. I've included an example of how to add skip behavior in the [README](https://github.com/braintree/runbook#tags-and-labels). The hook should be defined in you `Runbookfile` for it to apply to all runbooks across your project.
